webserver/errors.py

```diff
@@ -10,6 +10,8 @@ def api_error(error):
 
     @app.errorhandler(400)
     def bad_request(error):
+        if request.path.startswith('/api/v1/'):
```

Comment:
This should be a parameter shared with https://github.com/saifulbkhan/acousticbrainz-server/blob/61bae8d874be62d6ce2c2d5928056d2edb78b35f/webserver/__init__.py#L103